template/README.md

```diff
@@ -658,17 +673,15 @@ Create Elm App uses [elm-test](https://github.com/rtfeldman/node-test-runner) as
 
 To use packages in tests, you also need to install them in `tests` directory.
 
-Please note that you have to pass the path to the `elm-package.json` with your test dependencies.
-
 ```bash
-elm-app test --add-dependencies tests/elm-package.json
+elm-app test --add-dependencies elm-package.json
```

Comment:
This looks wrong to me, the path is supposed to point to `elm-package` [user wants to update.](https://github.com/rtfeldman/node-test-runner#--add-dependencies)

Reply:
This is because your fix isn't released yet (https://github.com/halfzebra/create-elm-app/commit/5d026a6c457729259ca9c045bcd193cc48f881ab). I started by actually creating a new elm app with `create-elm-app` in version `1.10.4`. So thanks a lot for having a detailed look.

Reply:
I added the change, and amended it to the commit.